config/routes.rb

```diff
@@ -33,6 +33,12 @@
       draw :listing
     end
 
+    namespace :authorization do
+      scope :articles do
+        get "create_post_button", controller: "articles"
+      end
+    end
```

Comment:
not in love with the structure/naming of this. as this grows, having this namespaced under "authorization" seems like it can get weird. maybe "components" or "parts" works better? ¯\_(ツ)_/¯ naming things is hard...

Reply:
I'll keep an eye on this organization as I move through the next batch of conditional renders.  A pattern will emerge.